src/schema/party/README.md

```diff
@@ -0,0 +1,25 @@
+# Party
+
+Parties are groups of connected players. They are used to join matchmaking
+together and during lobby balancing.
+
+A player can only be in at most one party at any given time.
+
+Any player in a party can invite any other connected player. When an invite is
+sent, the target player as well as every party member gets an event
+`party/invited`.
+
+Any pending invite can be cancelled by any member in the party with the request
+`party/cancelInvite`. Afterward, an event `party/inviteCancelled` will be sent
+to all party members and invited players.
```

Comment:
Invites by one member of the party will be cancellable by other party members? Not necessarily sure I have a problem with it, but it's atypical, and I'd like to hear some reasoning for it.